lib/rust/ensogl/core/src/display/shape/primitive/def/class.rs

```diff
@@ -206,6 +206,11 @@ where for<'t> &'t Self: IntoOwned<Owned = Self> {
         Shrink(self, value.into())
     }
 
+    /// Create an annulus of given thickness around shape's boundary.
```

Comment:
It'd be good to explain here what annulus is for people not knowing this term.

Reply:
If it is border, why don't we call it "border" as in CSS? If it's not a border, what's the difference?

Reply:
I'm going with "stroke", as it better describes what it does, while being recognizable.